packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -529,11 +529,11 @@ async function onDidStartDebugSessionHandler(event: vscode.DebugSession): Promis
   }
 }
 
-export function terminateAllRunningTeamsfxTasks(): void {
+export async function terminateAllRunningTeamsfxTasks(): Promise<void> {
   for (const task of allRunningTeamsfxTasks) {
     try {
       if (task[1] > 0) {
-        kill(task[1], "SIGTERM");
```

Comment:
Any need to add default TIMEOUT in case `killProcess` hangs?

Reply:
Yes, it makes sense.